enterprise/frontend/src/metabase-enterprise/embedding-sdk/index.js

```diff
@@ -0,0 +1,7 @@
+import { PLUGIN_EMBEDDING_SDK } from "metabase/plugins";
+import { hasPremiumFeature } from "metabase-enterprise/settings";
+
+// TODO: Change this to `embedding-sdk` once the flag is added
```

Comment:
This requires us to [add a new feature flag first.](https://www.notion.so/metabase/Feature-flag-and-settings-for-SDK-8ed424c776194528b6332c5fc2510a0c?pvs=4#df9e24fcaa754bdfbc19e0c42cdba66c)